### ISSUE TITLE
The new project contains a mismatched composer.lock

### ISSUE THREAD

**Issue by cbrunnkvist** [symfony/symfony-installer#172]
```
$ symfony new new-project ; cd new-project/ ; composer install
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Warning: The lock file is not up to date with the latest changes in composer.json. You may be getting outdated dependencies. Run update to update them.
```

I think it would be less confusing / make a less broken impression if it was at least in sync with itself for newly generated projects.

**Comment by Pierstoval** [comment]
It's an issue from [Symfony standard](https://github.com/symfony/symfony-standard), not from the installer. Actually, when creating a new project, you do not have to use `composer install` because vendors are already packaged in the zipball downloaded from `get.symfony.com`. 
What you can do, though, is a `composer update`, which will not throw any error.
But it's not that good if you are using Symfony 2.6 or 2.3 (because it will update to 2.7), so you'll have to update the dependency manually in your `composer.json` file.

**Comment by javiereguiluz** [comment]
@cbrunnkvist as @Pierstoval said, there is no need to do a `composer install` because the installer already does that for you. After finishing the execution of the `symfony new my_project` command, everything is ready to start developing your application.

You can do a `composer update` to get the latest dependency versions or `composer require ...` to install something, but there is no need for `composer install`.

**Comment by Pierstoval** [comment]
@javiereguiluz Technically, the installer does not "do that for you", because the vendors are installed with the script that generates the zipballs on `get.symfony.com` , IIRC :)

**Comment by cbrunnkvist** [comment]
@Pierstoval yes composer update should only be run in case your intent is _to get all updates for ALL dependencies_. So for future reference: the correct way to fix this warning is to regenerate the __lock file__ to be up to date with the json spec - and nothing more:

`compser update --lock`

Issue still remains - everything regarding dependencies should be in sync from the get-go. Who is it that bakes this elusive zip file for us?

**Comment by Pierstoval** [comment]
Actually, when you run composer update, the lock file updates automatically

**Comment by javiereguiluz** [comment]
OK, I think I know what's happening. After downloading the Symony contents, the installer [changes the composer.json file](https://github.com/symfony/symfony-installer/blob/master/src/Symfony/Installer/NewCommand.php#L62) to provide sensible defaults for some options. This is why `composer.json` and `composer.lock` are out of sync.

**Comment by javiereguiluz** [comment]
I'm trying to fix this issue in #173. Please review it. Thanks!
